### PR TITLE
Updating Docs

### PR DESCRIPTION
**Pull request by brittainhard**

### CODE REVIEW

File: docs/source/dev_guide.rst [hunk 1]
```diff
@@ -6,13 +6,9 @@ Developer's Guide to Memex Explorer
 Setting up Memex Explorer
 *************************
 
-To setup your machine, you will need Anaconda or Miniconda
-installed. Miniconda is a minimal Anaconda installation that
-bootstraps conda and Python on any operating system. Install `Anaconda
-<http://continuum.io/downloads>`_ or `Miniconda
-<http://conda.pydata.org/miniconda.html>`_ from their respective sites.
+To setup your machine, you will need Anaconda or Miniconda installed. Miniconda is a minimal Anaconda installation that bootstraps conda and Python on any operating system. Install `Anaconda <http://continuum.io/downloads>`_ or `Miniconda <http://conda.pydata.org/miniconda.html>`_ from their respective sites.
```
Comment (ahmadia) [inline docs/source/dev_guide.rst]: verb form is "set up", not setup.

**Comment by ahmadia** [comment]
Thanks @brittainhard - looks great.

File: docs/source/dev_guide.rst (hunk 1 above)
Comment (brittainhard) [inline docs/source/dev_guide.rst]: Are we in a grammar fight now @ahmadia?

Comment (ahmadia) [inline docs/source/dev_guide.rst]: I'll defer to your expertise, I think this is just a stale word bug that we inherited in the docs.

**Comment by ahmadia** [comment]
@brittainhard - please merge whenever you're ready.

**Comment by brittainhard** [comment]
This is only partly done. Gonna spend the rest of the day tomorrow rewriting all of it.

**Comment by ahmadia** [comment]
Sounds good.  Get some rest :)

File: docs/source/user_guide.rst [hunk 2]
```diff
@@ -9,23 +9,16 @@ constantly evolving as a result. For documentation requests, please*
...
+
+Web Crawling
+  A service to create, run, and analyze `Nutch <http://nutch.apache.org/>`_ and `ACHE <https://github.com/ViDA-NYU/ache>`_ crawls. The crawl operation is heavily abstracted and simplified. Users provide a list of seed URLs to start the crawl, and in the case of ACHE's targeted crawling, a machine learning `model <https://github.com/ViDA-NYU/ache#build-a-model-for-aches-page-classifier>`_ to declare the relevancy of crawled pages.
+
+Domain Discovery Tool
+  Through the use of `Domain Discovery Tool <https://github.com/ViDA-NYU/domain_discovery_tool>`_, the user can search for content in the web and build data models based on clustering algorithms. The user can search the web and highlight relevant and irrelevant pages, and DDT will produce data model files, which you can use with Ache crawls in Memex Explorer.
+
+DataWake
+   `DataWake <https://github.com/Sotera/Datawake>`_ is a server and firefox plugin that tracks your search investigations. It keeps track of where you search, so that "trails" can be built out of the information that you gather. These trails can be converted to seeds lists in Memex Explorer, and can be used in both...
```
Comment (brittainhard) [inline docs/source/user_guide.rst]: Have to get the wording right here. This is a first draft and incomplete.

**Comment by brittainhard** [comment]
This is ready for review.

**Comment by brittainhard** [comment]
Merging if no comment.

**Pull request merged**